source/tutorial/kerberos.txt

```diff
@@ -68,12 +70,24 @@ Kerberos authentication on either Windows or macOS.
                setspn.exe -A mongosql/BI.EXAMPLE.COM mongosql
```

Comment:
Using -S instead of -A prevents duplicates

Reply:
We can also put after the ``mongodb@EXAMPLE.COM`` and ``mongosql@EXAMPLE.COM`` that the actual name of these users is not important.

Reply:
Agreed. Fixed.